src/app/core/services/basic-rundown-state.service.ts

```diff
@@ -78,6 +79,16 @@ export class BasicRundownStateService implements OnDestroy {
     this.basicRundownsSubject.next(this.basicRundowns)
   }
 
+  private rehearsalBasicRundownFromEvent(event: RundownEvent): void {
```

Comment:
it should be `rehearse` not `rehearsal` here.
`rehearsal` is the noun, but we use verbs when naming methods, because methods _does something_.

Same for the `basicEntityService`

Reply:
Fixed